test/fixtures/functions-without-gatsby-plugin/.netlify/state.json

```diff
@@ -0,0 +1,3 @@
+{
+	"siteId": "3e2bfaf8-ea4e-4fef-8660-0267289dbc77"
+}
```

Comment:
interesting choice to commit this. i'm totally cool w it, we just gitignored it in next and require devs to run ntl init or put state.json there themselves 😬

Reply:
Yeah, it's only there so that `ntl build` will work in the tests! I don't _think_ it's sensitive information? There's no key included anywhere

Reply:
yeh there is no reason not to include this. identity is stored on the client that authorises the CLI